src/plugins/hetero/tests/functional/skip_tests_config.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "functional_test_utils/skip_tests_config.hpp"
+
+#include <string>
+#include <vector>
+
+#include "openvino/core/core_visibility.hpp"
+
+std::vector<std::string> disabledTestPatterns() {
+    std::vector<std::string> retVector{
+        R"(.*smoke_(Multi|Auto|Hetero)_BehaviorTests.*OVPropertiesTests.*SetCorrectProperties.*)",
```

Comment:
at first glance, why set correct property not support for multi/auto/hetero?
I suppose we can not set for HETERO:HWDEVICE, but we should be able to set for HETERO/MULTI/AUTO themselves right?
cc @yangwang201911

Reply:
TODO: CVS-140061